doc/classes/@GDScript.xml

```diff
@@ -348,6 +394,15 @@
 			<argument index="1" name="funcname" type="String">
 			</argument>
 			<description>
+				Returns a reference to the specified function 'funcname' in object 'instance'.
+				[codeblock]
+				a = funcref(self, "foo")
+				a.call_func()
+
+				func foo():
+				    print("bar")
+				[/codeblock]
+				Prints out "bar". This is a trivial example, the real power is that variable "a" can be passed around to functions.
```

Comment:
I'd remove this and explain the purpose at the top - it's very important info - funcref "turns functions into objects" (produces a reified method)